.eslintrc.json

```diff
@@ -9,7 +9,8 @@
         "airbnb", 
         "airbnb/hooks", 
         "airbnb-typescript", 
-        "prettier"
+        "prettier",
+        "react-app"
```

Comment:
Heads up `"airbnb"` has support for react so `"react-app"` is going to overwrite a lot of those.

Reply:
Just removed.